app/models/service.rb

```diff
@@ -85,8 +85,47 @@ def deployment_finished?(deployment_id)
     backend.deployment_finished?(deployment_id)
   end
 
+  def save_and_update_container_count!(desired_container_count)
+    self.desired_container_count = desired_container_count
+    self.save!
+
+    ecs.update_service({
+      desired_count: desired_container_count,
+      cluster: district.name,
+      service: logical_name
+    })
+  end
```

Comment:
This will fail if `desired_container_count` is greater than the max of the service.
I think it's better to check the result and raise an exception on failure.

It's ok if you can detect it by the output of `bcn` command.

Reply:
<img width="512" alt="スクリーンショット 2022-07-08 10 58 53" src="https://user-images.githubusercontent.com/874280/177901409-9d90813b-5b00-4faf-9af0-37023aa19c4f.png">
 
I attempted to find the max count but I could not. I am not sure it exists

Reply:
Ah, I confused it with ASG, ECS service doesn't have max.